### DISCUSSION TITLE
Hi ! What is this mail ?

### DISCUSSION THREAD

**Discussion by BeingTraveler** [community/community#101820] [GitHub Education] [answered]
### Hi everyone,

What is the mean of this mail from Github ?
![image](https://github.com/community/community/assets/139610939/450495cc-2f12-44c1-8a8e-b9c267747459)

**Comment by GaganHonor** [comment, marked as answer]
Oh ! As written in mail just follow simple guide lines :) 
or follow https://github.com/settings/two_factor_authentication/setup/intro or https://github.com/settings/security

This is neccessary to protect your account !


Thanks 
>Good Luck

**Comment by BeingTraveler** [comment]
🙏🙏
